docs/docs-beta/docs/guides/quality-testing/asset-checks.md

```diff
@@ -1,4 +1,54 @@
 ---
 title: "Test assets with Asset Checks"
 sidebar_position: 10
----
\ No newline at end of file
+---
+
+Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets directly in Dagster.
+
+This guide covers the most common use cases for testing assets with asset checks and taking action based on the result.
+
+<details>
+<summary>Prerequisites</summary>
+- Familiarity with [Assets](/concepts/assets)
+</details>
+
+## Testing assets with a single asset check
+
+The example below defines a single asset check on an asset that fails if the `order_id` column of the asset contains a null value.
+
+In this example, the asset check will run after the asset has been materialized, to ensure the quality of its data.
+
+<CodeExample filePath="guides/data-assets/quality-testing/asset-checks/single-asset-check.py" language="python" title="Asset with a single asset check" />
+
+## Testing assets with multiple asset checks
+
+In most cases, checking the data quality of an asset will require multiple checks.
+
+The example below defines two asset checks on an asset using the `multi_asset_check` decorator.
+- one that fails if the `order_id` column of the asset contains a null value.
+- another one that fails if the `item_id` column of the asset contains a null value.
+
+In this example, both asset check will run in a single operation after the asset has been materialized.
+
+<CodeExample filePath="guides/data-assets/quality-testing/asset-checks/multiple-asset-checks.py" language="python" title="Asset with multiple asset checks" />
+
+## Blocking downstream assets
+
+If an asset check fails, it may be useful to block the materialization of downstream assets to prevent the problem from spreading. To enable this behavior, set the `blocking` argument to `True` in the `asset_check` decorator.
```

Comment:
```suggestion
By default, materialization of downstream assets will continue, even if a parent's asset check fails. To block the materialization of downstream assets, set the `blocking` argument to `True` in the `asset_check` decorator.
```

I think emphasizing that it doesn't block "by default" is good

Reply:
Done in [e241f07](https://github.com/dagster-io/dagster/pull/24035/commits/e241f0776dea8f3c50bbee02c23173b6819e155f)